edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Edgelet/versioning/ModuleManagementHttpClientVersioned.cs

```diff
@@ -103,6 +108,12 @@ public virtual async Task<Stream> GetModuleLogs(string module, bool follow, Opti
 
         protected abstract void HandleException(Exception ex, string operation);
 
+        bool IsModuleManagementSocketStale(Exception ex)
+        {
+            return (string.Compare(ex.GetType().ToString(), "System.Net.Internals.SocketExceptionFactory+ExtendedSocketException") == 0)
+                && ex.Message.ToString().Contains("Connection refused");
```

Comment:
```suggestion
                && ex.Message.Contains("Connection refused");
```

Reply:
add case insensitive contains in stringex.cs https://dotnetthoughts.net/how-to-make-string-contains-case-insensitive/, and use here.

Reply:
I think it should be the exact match, error message won't simply change the casing for itself.